hail/python/hail/docs/makeDocs.sh

```diff
@@ -17,8 +17,19 @@ xsltproc --html -o build/www/jobs.html www/jobs.xslt build/tmp/jobs.html
 pandoc -s www/about.md -f markdown -t html --mathjax --highlight-style=pygments --columns 10000 -o build/tmp/about.html
 xsltproc --html -o build/www/about.html www/about.xslt build/tmp/about.html
 
+cleanup() {
+    set +e
+    rm -f python/hail/docs/change_log.rst
+}
+trap cleanup EXIT
```

Comment:
I prefer the cleanup function to be at the top of the script because we can only have one handler per signal per script.

Also, the interpretation of traps by different shells [is horrendously discordant](https://unix.stackexchange.com/questions/240723/exit-trap-in-dash-vs-ksh-and-bash). I think
```
trap cleanup EXIT
trap 'exit 1' INT TERM
```
Works in the vast majority of cases for the vast majority of shells.